packages/plugins/experimental/plugin-calls/src/utils/cf-calls-peer.ts

```diff
@@ -0,0 +1,429 @@
+//
+// Copyright 2024 DXOS.org
+//
+
+import { PersistentLifecycle } from '@dxos/async';
+import { type Context, Resource } from '@dxos/context';
+import { invariant } from '@dxos/invariant';
+import { log } from '@dxos/log';
+
+import { BulkRequestDispatcher, FIFOScheduler } from './util';
+import { type RenegotiationResponse, type TrackObject, type TracksResponse } from '../types';
+
+export type CloudflareCallsConfig = {
```

Comment:
CallsConfig (no need for CF, in names unless it really disambiguates things).
Maybe CallsServiceConfig. Same in other places and filename.